.github/workflows/run-bundle-test.yml

```diff
@@ -2,6 +2,8 @@ name: Run bundle test
 
 on:
   push:
+    branches-ignore:
+      - develop # the daily release reaches GitHub before PyPI, and these fail in that window
```

Comment:
We should also add `master` to this list right?

Reply:
Added via https://github.com/aws/aws-cli/pull/9471/commits/80a12258718206484e0493cd2a175fedec6b9aaa, forgot we still push there.